Mention sanitization with example in compatibility section (#4692)
Particular/docs.particular.net: transports/azure-service-bus/compatibility.md

### Namespace alias is not used

The Azure Service Bus transport doesn't support [namespace aliases](/transports/azure-service-bus/legacy/securing-connection-strings.md).

### Sanitization rules might need to be aligned

If the usage of the legacy transport involved [sanitization](/transports/azure-service-bus/legacy/sanitization.md) of entity names the sanitization logic needs to be made compatible.

> For example, if the `ValidateAndHashIfNeeded` strategy is used the sanitization functions need to be adjusted to create the same entity names.

Reviewer: ```suggestion
For example, if the `ValidateAndHashIfNeeded` strategy was used then the sanitization functions need to include the strategy logic to presrve the same entity names.
```
